### PR TITLE
Add `durumis.com`

### PR DESCRIPTION
**Pull request by calmlake79**
Public Suffix List (PSL) Pull Request (PR) Template
====

Each PSL PR needs to have a description, rationale, indication of DNS validation and syntax checking, as well as a number of acknowledgements from the submitter.  This template must be included with each PR, and the submitting party MUST provide responses to all of the elements in order to be considered.

### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the \_PSL txt record in place in the respective zone(s) in the affected section

__Submitter affirms the following:__ 

  * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)
    - [Cloudflare](https://developers.cloudflare.com/learning-paths/get-started/add-domain-to-cf/add-site/)
    - [Letsencrypt](https://letsencrypt.org/docs/rate-limits/)
    - <UPDATE THIS LIST WITH YOUR LIMITATIONS!>

  * [x] This request was _not_ submitted with the objective of working around other third-party limits

  * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting

---

For Private section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x]  *Yes, I understand*.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  *Proceed*.
---

Description of Organization
====

[https://durumis.com/](https://durumis.com/)

I am the owner and operator of durumis, a blog platform that allows users to create and host their own blogs easily. I am requesting this entry in the PSL in order to enable this functionality.

Reason for PSL Inclusion
====

durumis is a blog platform that allows users to create and host their own blogs. To allow users to register subdomains for their own blogs, we need an entry in the PSL. This will allow us to issue certificates and cookies for user subdomains in a secure manner.

The durumis.com domain has a remaining registration term of over 9 years. We will maintain the _PSL TXT record in place in the respective zone(s).

Number of users this request is being made to serve:
The current number of users on the platform is approximately 5,000, and the number is expected to grow rapidly.

DNS Verification via dig
=======

```
dig +short TXT _psl.durumis.com
"https://github.com/publicsuffix/list/pull/1978"
```

Results of Syntax Checker (`make test`)
=========
```
PASS: libpsl_icu_load_dafsa_fuzzer
PASS: libpsl_icu_fuzzer
PASS: libpsl_icu_load_fuzzer
============================================================================
Testsuite summary for libpsl 0.21.5
============================================================================
# TOTAL: 3
# PASS:  3
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
Making check in tests
  CC       test-is-public.o
  CC       common.o
  CC       test-is-public-all.o
  CC       test-is-cookie-domain-acceptable.o
  CC       test-is-public-builtin.o
  CC       test-registrable-domain.o
  CCLD     test-is-public
  CCLD     test-is-cookie-domain-acceptable
  CCLD     test-is-public-all
  CCLD     test-is-public-builtin
  CCLD     test-registrable-domain
PASS: test-is-public-builtin
PASS: test-is-public
PASS: test-is-cookie-domain-acceptable
PASS: test-registrable-domain
PASS: test-is-public-all
============================================================================
Testsuite summary for libpsl 0.21.5
============================================================================
# TOTAL: 5
# PASS:  5
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
Making check in msvc
```

### CODE REVIEW

File: public_suffix_list.dat [hunk 1]
```diff
@@ -7844,6 +7844,11 @@ dupont
 // https://www.iana.org/domains/root/db/durban.html
 durban
 
+// durumis : https://durumis.com
+// Dreamyoungs, Inc.
+// Submitted by Harrison Jung <harrisonjung@dreamyoungs.com>
```
Comment (simon-friedberger) [inline public_suffix_list.dat:7849]: Use a non-personal e-mail address.

File: public_suffix_list.dat [hunk 2]
```diff
@@ -7844,6 +7844,11 @@ dupont
 // https://www.iana.org/domains/root/db/durban.html
 durban
 
+// durumis : https://durumis.com
```
Comment (simon-friedberger) [inline public_suffix_list.dat:7847]: Use a single line for the organization name and sort by that name.

**Comment by simon-friedberger** [comment]
* [x] Expiration (Note: Must STAY >2y at all times)
  * [x]  durumis.com expires 2033-09-26
* [x] DNS _psl entries (Note: Must STAY in place)
* [x] Tests pass
* [ ] Sorting
* [ ] Reasoning/Organization description

**Comment by simon-friedberger** [comment]
durumis.com returns code 500.

**Comment by calmlake79** [comment]
I have made the changes you suggested. 
- Use non-personal e-mail. ( cto@dreamyoungs.com )
- Combinded Line.  

However, it seems there's an error occurring during the build process in GitHub Actions. 
Could you please advise on how to troubleshoot this issue?

**Comment by calmlake79** [comment]
> durumis.com returns code 500.

I have made the changes you suggested. 
- Use non-personal e-mail. ( cto@dreamyoungs.com )
- Combinded Line.  
- 500 Error.

However, it seems there's an error occurring during the build process in GitHub Actions. 
Could you please advise on how to troubleshoot this issue?

**Comment by calmlake79** [comment]
PSL Registration Request (durumis.com): Additional Information
**Public Suffix List Registration Request**: durumis.com

**Reason for Request**:

durumis.com is a blog platform operated by Dreamyoungs Inc., a South Korean company. It provides users with personalized subdomains (e.g., user-id.durumis.com ) to help them build their own brands and create unique blogs.

We expect the following benefits from PSL registration:

- **Browser Compatibility**: Ensure that all major web browsers can correctly recognize and process durumis.com subdomains.
- **Enhanced Security**: Protect users from phishing, cookie theft, and other attacks, and provide a safe environment for blogging.
- **Search Engine Optimization (SEO)**: Increase the credibility of durumis.com subdomains in search engines, positively impacting search result rankings.

**About Dreamyoungs Inc.**:

Dreamyoungs Inc. is a South Korean IT company that strives to provide user-centric services. durumis.com is one of Dreamyoungs Inc.'s core services, aiming to provide a space where users can freely express their thoughts and communicate.

Thank you for your consideration. We look forward to a positive response regarding the PSL registration.

**Comment by yahesh** [comment]
The location of this domain is wrong. It has been added to the `ICANN DOMAINS` section of the PSL. Please move it to the correct place in the `PRIVATE DOMAINS` section of the PSL.

**Comment by calmlake79** [comment]
> The location of this domain is wrong. It has been added to the `ICANN DOMAINS` section of the PSL. Please move it to the correct place in the `PRIVATE DOMAINS` section of the PSL.
> 
> Furthermore, the reason for the request is to _"issue certificates [...] for user subdomains in a secure manner"_. This sounds as if the addition to the PSL is meant to circumvent limitations of 3rd parties.

@yahesh //
You are absolutely right; mentioning "issuing certificates" appears to have been an oversight, likely introduced during the translation process. Thank you for bringing this to my attention!

The core issue lies in our commitment to providing personalized subdomains (e.g., user-id.durumis.com) for our users' blogs. We believe this feature empowers our users to cultivate their unique online identities. However, we've encountered unforeseen complexities related to cookie handling on these subdomains.

After thorough research, we are convinced that adding durumis.com to the PSL is the most effective solution to ensure a seamless and secure browsing experience. We are dedicated to providing the best possible platform for our users, and resolving these technical challenges is paramount to achieving that goal. We kindly request your approval for this change so we can continue to improve our platform and better serve our users.

**Comment by simon-friedberger** [comment]
@calmlake79 Sorting is still wrong. Can you provide some examples of sites? Have you read the disclaimer about potential impact on cookies for durumis.com?

**Comment by calmlake79** [comment]
@simon-friedberger //

We are writing to request a Public Suffix List (PSL) entry for our domain, "durumis.com".

Currently, we are piloting a system where we issue subdomains to our users in stages.
Here are a few examples:

https://official.durumis.com  
https://ai-news-japan.durumis.com  
https://rebeka.durumis.com  

However, due to our absence from the PSL, we are facing some security limitations that restrict us from providing certain services.

One notable example is cookie management. Our development team needs the ability to freely define policies for cross-subdomain cookie access. Ideally, we need to control which cookies can be accessed by other subdomains from a given subdomain. However, we are encountering obstacles in implementing these controls effectively.

For instance, we need to prevent a scenario where a cookie set by "sub1.durumis.com" can be arbitrarily read by "sub2.durumis.com", as this could pose a privacy risk. Therefore, we require the ability to configure these settings appropriately.

We believe that adding "durumis.com" to the PSL would grant us the necessary control over our subdomains and enable us to offer a secure and robust user experience.

Thank you for your time and consideration. We look forward to your positive response.

( Sorting is updated! Thank you! )

**Comment by wdhdev** [comment]
@calmlake79 Please place the `_psl.durumis.com` TXT record back in order to prevent a future removal, thank you. These are required to stay in place.

**Pull request merged**